lib/compilers/hook.js

```diff
@@ -0,0 +1,24 @@
+import path from 'path';
```

Comment:
Can you make this a typescript file? The changes should be minimal other than extension renaming

Reply:
Now that there's a ts file, this one is no longer needed

Reply:
Ok.